packages/app-desktop/commands/exportDeletionLogs.ts

```diff
@@ -0,0 +1,42 @@
+
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import Logger from '@joplin/utils/Logger';
+import { TargetType } from '@joplin/utils/Logger';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import bridge from '../services/bridge';
+
+export const declaration: CommandDeclaration = {
+	name: 'exportDeletionLogs',
+	label: () => _('Export deletion logs'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async () => {
+			if (!Logger.globalLogger) {
+				throw new Error('Could not find global logger');
+			}
+
+			const fileTarget = Logger.globalLogger.targets().find(t => t.type === TargetType.File);
+
+			if (!fileTarget || !fileTarget.path) {
+				throw new Error('Log file target not found');
+			}
+
+			const logFile: string = await shim.fsDriver().readFile(fileTarget.path);
+
+			const deletionLog = logFile
+				.split('\n')
+				.filter(line => line.includes('DeleteAction'))
+				.join('\n');
+
+			const deletionLogPath = `${Setting.value('profileDir')}/deletion_log.txt`;
+
+			await shim.fsDriver().writeFile(deletionLogPath, deletionLog, 'utf8');
+
+			await void bridge().openItem(Setting.value('profileDir'));
```

Comment:
Instead please open the file you've just created. Seems a bit random to open the profile directory as the user won't know why we're doing this

Reply:
Also no `await void` please